web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/course/CourseResource.java

```diff
@@ -231,6 +223,7 @@ public Page list(@PathParam("id") Long id, @PathParam("resource") String resourc
 	@GET
 	@Produces({ MediaType.APPLICATION_JSON })
 	@Path("list")
+	@ApiOperation(value="list",hidden = true)
 	public CourseListIndex listIndex() throws Exception {
```

Comment:
is there an annotation to exclude this one from swagger.json?

Reply:
The `@ApiOperation(...hidden = true)` _is_ the annotation to "exclude this one from swagger.json"

Reply:
ah sry, overlooked that - remembered that .yaml only (which still contained it)